features/expectations/body_json_example.feature

```diff
@@ -33,7 +33,7 @@ Feature: Body - JSON example
       "string": "Hello World"
     }
     """
-    Then Gavel will set some error for "body"
+    Then field "body" is NOT valid
```

Comment:
As soon as I read this with capital NOT I couldn't stop thinking about it missing SHOULD/MUST/SHALL from [RFC2119](https://tools.ietf.org/html/rfc2119), perhaps something to consider using:

```suggestion
    Then field "body" SHALL NOT be valid
```

Reply:
🤔That's a good idea, thanks! I've created an issue to consider this: https://github.com/apiaryio/gavel-spec/issues/47

I don't think this is a blocking change for the current pull request.

Reply:
Yes certainly wasn't meant to block this PR. All but the two comments I mentioned on Slack are more of thinking after first time looking at gavel-spec.